docs/source/guide/tutorials/base/how_to_train/instance_segmentation.rst

```diff
@@ -43,40 +43,76 @@ environment:
 Dataset preparation
 ***************************
 
-1. Let's use the simple toy dataset `Car, Tree, Bug dataset <https://github.com/openvinotoolkit/training_extensions/tree/develop/tests/assets/car_tree_bug>`_
-provided by OpenVINO™ Training Extensions.
+..  note::
 
-This dataset contains images of simple car, tree, bug with the annotation for instance segmentation.
+  Currently, we support the following instance segmentation dataset formats:
 
-- ``car``	- Car Shape Illustration
-- ``tree``	- Tree Shape Illustration
-- ``bug``	- Bug Shape Illustration
+  - `COCO <https://cocodataset.org/#format-data>`_
 
-This allows us to look at the structure of the dataset used in instance segmentation, and can be a good starting point for how to start an instance segmentation task with OpenVINO™ Training Extensions.
 
+1. Clone a repository with 
+`WGISD dataset <https://github.com/thsant/wgisd>`_.
 
-.. image:: ../../../../../utils/images/car_tree_bug_gt_sample.png
-  :width: 400
+.. code-block::
+
+  mkdir data ; cd data
+  git clone https://github.com/thsant/wgisd.git
+  cd wgisd
+  git checkout 6910edc5ae3aae8c20062941b1641821f0c30127
+
+
+This dataset contains images of grapevines with the annotation for different varieties of grapes.
+
+- ``CDY`` - Chardonnay
+- ``CFR`` - Cabernet Franc
+- ``CSV`` - Cabernet Sauvignon
+- ``SVB`` - Sauvignon Blanc
+- ``SYH`` - Syrah
+
+|
+
+.. image:: ../../../../../utils/images/wgisd_dataset_sample.jpg
+  :width: 600
+  :alt: this image uploaded from this `source <https://github.com/thsant/wgisd/blob/master/data/CDY_2015.jpg>`_
 
+|
 
 2. Check the file structure of downloaded dataset,
 we will need the following file structure:
 
 .. code-block::
 
-  car_tree_bug
+  wgisd
   ├── annotations/
-    ├── instances_train.json
-    └── instances_val.json
+      ├── instances_train.json
+      ├── instances_val.json
+      (Optional)
+      └── instances_test.json
   ├──images/
-    └── <images>
-  ...
+      (Optional)
```

Comment:
Is this right place for `(Optional)`?

Reply:
Yes, this is not a requirement when training.